src/mappings/nfts/getters/statemint.ts

```diff
@@ -276,7 +276,7 @@ function getSetAttributeEvent(ctx: Context): SetAttribute {
       collectionId: classId.toString(),
       sn: instanceId?.toString(),
       trait: key.toString(),
-      value: value.toString(),
+      value,
```

Comment:
And what about other values? are they gonna be decoded properly?

Reply:
I'll try to rephrase:
Are the other `values` set using `setAttribute` call alright?

Reply:
See screenshot, Entity `14-3` royalty is set using a string.
Didn't tested other attribs.